static/js/invite.js

```diff
@@ -70,14 +70,35 @@ function submit_invitation_form() {
                 // Some users were not invited.
                 var invitee_emails_errored = [];
                 var error_list = [];
+                var is_invitee_deactivated = false;
+
                 arr.errors.forEach(function (value) {
                     error_list.push(value.join(': '));
+                    if (value[1] === "Account has been deactivated.") {
```

Comment:
This check won't work correctly if the user is using a non-English language; in that case, the string will be a translation.  We should be checking something from the API that's clear.